lib/test/cdo/test_slack.rb

```diff
@@ -46,12 +48,57 @@ def test_update_topic_with_error_response
   def test_message
     CDO.stubs(:slack_endpoint).returns('fake-slack-endpoint')
     Net::HTTP.expects(:post_form).returns(stub(code: 200))
-    assert Slack.message(FAKE_CHANNEL, channel: FAKE_MESSAGE)
+    assert Slack.message(FAKE_MESSAGE, channel: FAKE_CHANNEL)
+  end
+
+  def test_message_uses_channel_map
```

Comment:
These tests were borrowed from `shared/test/test_hip_chat.rb`. That file is being deleted in https://github.com/code-dot-org/code-dot-org/pull/13519.